src/platform/packages/shared/kbn-esql-validation-autocomplete/src/autocomplete/autocomplete.ts

```diff
@@ -281,7 +277,16 @@ export async function suggest(
       getPolicyMetadata
     );
   }
-  return [];
+
+  const multiWordKeywordMatchRange = getMultiWordKeywordMatchRange(innerText);
+
+  if (multiWordKeywordMatchRange) {
+    suggestions.forEach((s) => {
+      s.rangeToReplace = multiWordKeywordMatchRange;
+    });
+  }
```

Comment:
This part handles the case where we detect a match on a multi-word keyword such as `IS NULL`. Monaco won't do this for us because by it is always going to treat spaces as word separators (as it should) unless we attach an explicit prefix range.